### ISSUE TITLE
Windows 10/11: submit the NVDA installer executable to revamped Microsoft Store

### ISSUE THREAD

**Issue by josephsl** [nvaccess/nvda#13058]
Hi,

Brought up on two issues:

### Background and rationale
Windows 8 (2012) introduced Microsoft Store (formerly known as Windows Store) to serve as a distribution platform for packaged apps utilizing Windows Runtime (WinRT). Over the years, Store has become a venue for distributing not only WinRT and later UWP (Universal Windows Platform) apps, but also a marketplace for movies and books. Later, Microsoft allowed developers to package classic desktop apps in an appx bundle for distribution on Microsoft Store, which NVDA does take advantage of with limitations such as no add-ons, no code injection in web browsers, among other things.

In June 2021, as part of Windows 11 introduction, Microsoft changed Store policies. Among these changes is the ability to publish classic desktop apps without going through Desktop App Converter i.e. packaging it inside an appx bundle. With this change, it becomes possible to publish the NVDA installer and other desktop apps, the best known example is desktop version of Firefox web browser.

### Is your feature request related to a problem? Please describe.
Currently the Store version of NVDA comes with limitations, including no add-ons, no code injection in web browsers, among others. This is more toward limitations on converted desktop apps as Microsoft envisioned Windows API developers to embrace UWP more. All this changed with Store policy change.

### Describe the solution you'd like
Publish NVDA installer on Microsoft Store to take advantage of the policies described in this Windows Blogs post:
https://blogs.windows.com/windowsdeveloper/2021/09/28/microsoft-store-more-apps-more-open/

Note that this applies to Windows 11, but a Microsoft employee tweeted that the new Store will be backported to Windows 10. Indeed, the tweet mentions that the revamped Store is rolling out to Windows 10 users as of the time of this writing.

### Describe alternatives you've considered
Keep distributing Store version of NVDA and/or advertise the download link for NvDA installer.

### Additional context
This change affects:

* Windows 10 Version 1709 or later, ideally Version 2004 or later
* Windows 11

The new Windows 10 Store is identical to Windows 11 Store except Windows 10 does not have Windows Subsystem for Android and therefore only Windows 11 users will be able to instlal Android paps from Microsoft Store via Amazon Appstore.

Thanks.

**Comment by Adriani90** [comment]
NV Access stated the capacity to handle the Microsoft Store in parallel to the usual version of NVDA is limited, so it is currently very low priority. How ever, the decision can be revisited when appropriate. I am converting this to a discussion.
